.github/workflows/build.yml

```diff
@@ -16,14 +16,29 @@ on:
 jobs:
     tests:
         runs-on: ubuntu-latest
-        name: "PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, ORM ${{ matrix.orm }}"
+        name: >
+            PHP ${{ matrix.php }},
+            Symfony ${{ matrix.symfony }},
+            ORM ${{ matrix.orm }}
+            ${{ matrix['behat-transliterator'] != '' && matrix['gedmo-doctrine-extensions'] != ''
+                && format('(with Behat transliterator {0} & Gedmo {1})', matrix['behat-transliterator'], matrix['gedmo-doctrine-extensions'])
+                || '' }}
         strategy:
             fail-fast: false
             matrix:
                 orm: ['2.*', '3.*']
                 php: ["8.2", "8.3"]
+                behat-transliterator: [""]
                 composer-flags: ['--no-scripts --prefer-stable --prefer-dist']
+                gedmo-doctrine-extensions: [""]
                 symfony: ["6.4.*", "7.4.*"]
+                include:
+                    # with Behat transliterator and Gedmo Doctrine extensions optional packages
```

Comment:
I started here the new way of handle optional packages.
I plan to remove all of them from dev requirements and add tests with them.
This way, we are sure they are really optional. Moreover we are able to support a new version of Symfony without them to be ready.